### DISCUSSION TITLE
how to send bluetooth packets?

### DISCUSSION THREAD

**Discussion by tigercoding56** [wasp-os/wasp-os#394] [Q&A] [answered]
i would like to write an application to allow me to use the pinetime smartwatch as a motion controller of sorts 
for that purpose i want to send the orientation of the watch to my computer using bluetooth  , but i can't find documentation on how bluetooth works / is implemented in wasp-os    , so how would i do that ?

**Comment by fgaz** [comment, marked as answer]
See https://github.com/wasp-os/wasp-os/issues/374 https://github.com/wasp-os/wasp-os/issues/347 https://github.com/wasp-os/wasp-os/issues/332 https://github.com/wasp-os/wasp-os/issues/110

**Reply by tigercoding56** [reply]
Ty
